charts/daemonset/templates/daemonset.yaml

```diff
@@ -99,6 +125,5 @@ spec:
         {{- range $v := .Values.hostPaths }}
         - mountPath: "{{ $v.mountPath }}"
           name: {{ $v.name }}
-          readOnly: true
+          readOnly: {{ $v.readOnly }}
```

Comment:
Might be nice to default to "true" if `readOnly` is not supplied in the configuration block?  This means you could update the chart from old version to this one without seeing a change in behavior?